recipes/preseq/meta.yaml

```diff
@@ -1,16 +1,14 @@
 package:
   name: preseq
-  version: 2.0.2
+  version: 2.0.3
 
 source:
-  url: https://github.com/smithlabcode/preseq/releases/download/v2.0.2/preseq_v2.0.2.tar.bz2
-  fn: preseq_v2.0.2.tar.bz2
-  md5: 9f2a7b597c9f08b821db6ee55e2ea39c
-  patches:
-    - makefile.patch
+  git_url: https://github.com/smithlabcode/preseq
```

Comment:
Is there any reason you can not use the release tarball? https://github.com/smithlabcode/preseq/releases/download/v2.0.3/preseq_v2.0.3.tar.bz2
It seems the submodules are included.

Reply:
When I tried that it didn't seem to work, but I'll give it a whirl again.

Reply:
I must have used the master branch tarball before.